packages/graph/hash_graph/lib/graph/src/api/rest/mod.rs

```diff
@@ -462,6 +462,14 @@ impl Modify for TimeSchemaAddon {
             components
                 .schemas
                 .insert("Timestamp".to_owned(), Timestamp::<()>::schema());
+            components.schemas.insert(
+                "NullableTimestamp".to_owned(),
+                ObjectBuilder::new()
+                    .schema_type(SchemaType::String)
+                    .format(Some(SchemaFormat::KnownFormat(KnownFormat::DateTime)))
```

Comment:
I suppose it's intentional that we are using a `string` with a `DateTime` format, I would have just thought the client generator would create better types from this :laughing: 

Just to clarity `Timestamp` is also just `string` in the output, right?

Reply:
Yeah, I hoped that as well 😢 

Yes, `TimeStamp` is defined the same way, but without `.nullable`. In fact, I copied over their implementation. Which makes me wonder if I can reuse it somehow.

Reply:
Sadly, this would generate `type NullableTimeinterval = string`, so it's probably best to just keep the current approach